streams/catalog/src/main/java/com/hortonworks/streamline/streams/catalog/configuration/ConfigFileType.java

```diff
@@ -22,12 +22,12 @@
  * @see ConfigFileWriter
  */
 public enum ConfigFileType {
-  HADOOP_XML, PROPERTIES, YAML;
+  HADOOP_XML, PROPERTIES, YAML, ZOOKEEPER_CFG;
 
   public static ConfigFileType getFileTypeFromFileName(String fileName) {
     String fileExt = Files.getFileExtension(fileName);
     if (fileExt.isEmpty()) {
-      return null;
+      throw new IllegalArgumentException("Not supported config file extension: " + fileExt);
```

Comment:
This is a good option. You can also consider IllegalStateException, or create something like InvalidConfigException, if we don't already have something similar to it in the Streamline codebase.